File: app/src/main/java/com/firstlink/duo/model/Pager.java
package com.firstlink.duo.model;

import com.google.gson.annotations.SerializedName;

/**
 * Created by wzq on 15/6/2.
 */
public class Pager {

    @SerializedName("start_row")
    private int startRow;

    @SerializedName("page_size")
    private int pageSize;

    private int total;

    public int getStartRow() {
        return startRow;
    }

    public void setStartRow(int startRow) {
        this.startRow = startRow;
    }

    public int getPageSize() {
        return pageSize;
    }

    public void setPageSize(int pageSize) {
        this.pageSize = pageSize;
    }

    public int getTotal() {
        return total;
    }

    public void setTotal(int total) {
        this.total = total;
    }
}

File: app/src/main/java/com/firstlink/duo/model/Pager.kt
package com.firstlink.duo.model

import com.google.gson.annotations.SerializedName

/**
 * Created by wzq on 16/1/3.
 */
data class Pager(
        @SerializedName("start_row")
        val startRow: Int = 0,
        @SerializedName("page_size")
        val pageSize: Int = 0,
        val total: Int = 0
)